l10n/es.js

```diff
@@ -0,0 +1,17 @@
+OC.L10N.register(
+    "facerecognition",
+    {
+    "Persons" : "Personas",
+    "Looking for faces in this image..." : "Buscando por rostros en esta imagen...",
+    "Please enter a name to rename the person" : "Por favor ingrese un nombre para renombrar la persona",
+    "Rename" : "Renombrar",
+    "Rename Person" : "Renombrar Persona",
+    "Cancel" : "Cancelar",
+    "Face Recognition" : "Reconocimiento Facial",
+    "A face recognition app" : "Una aplicacion de reconocimiento facial",
+    "Open Documentation" : "Abrir Documentación",
+    "Configuration information" : "Información de la configuración",
+    "Here you can see the photos of your friends that we have recognized" : "Aqui puedes ver las fotos de tus amigos que reconocimos",
```

Comment:
"the photos" -> "photos".

Also, I would refrain from "we" and use passive voice, something like "that are recognized/that have been recognized". #Consider